addons/web_editor/static/src/js/wysiwyg/widgets/link.js

```diff
@@ -74,6 +74,7 @@ const Link = Widget.extend({
         if (this.data.range) {
             this.$link = this.$link || $(OdooEditorLib.getInSelection(this.editable.ownerDocument, 'a'));
             this.linkEl = this.$link[0];
+            this.linkEl.querySelectorAll('[data-o-link-zws]').forEach(zws => zws.remove());
```

Comment:
@SergeBayet Copying what `_resetLinkZws` does seems really louch. At minimum it should be documented. I am not sure how the zws feature work but in any case: modifying the link element when you initialize a widget (the link dialog here) can never be right. Also you'll see that `_resetLinkZws` is actually disabling observer before doing that, no reason it would not have to be done here too if this was the right solution.

What is the reason behind doing this instead of what you did previously?